examples/c++-api/Makefile

```diff
@@ -1,10 +1,22 @@
-all: count-demo
+CFLAGS=--std=c++11 \
+       -I ../../lib \
+	   -I ../../third-party/smhasher \
+	   -I ../../third-party/seqan/core/include/
```

Comment:
Is there a way (without too much pain) to make the example so that it doesn't use relative paths and relies on having run `make` fror khmer before? Thinking that as a real user I probably have my code in a different directory and want to somehow use the "installed" version of khmer instead of having my stuff in a subdirectory of khmer.